.readthedocs.yaml

```diff
@@ -0,0 +1,20 @@
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+version: 2
+
+sphinx:
+  builder: html
+  configuration: doc/source/conf.py
+  fail_on_warning: true
+
+# Set the version of Python and requirements required to build your docs
+python:
+  version: 3.8
```

Comment:
There is a new way to specify Python now: https://github.com/astropy/astropy/blob/1bf53dcf5672d5808580339725e301a587185d73/.readthedocs.yml#L8

Reply:
done!